debug.py

```diff
@@ -0,0 +1,24 @@
+from traceback import print_exc
+
+
+
+try: __debugload__
+except:   
+    __debugload__ = True
+    
+    debug = False
+    
+    def printdebug(str, msgdebug):
+        global debug
+        if msgdebug == False:
+            print(str)
+            return
+        
+        if msgdebug == True and debug == True:
+            print(str)
+            return
+            
+    def setdebug(set):
+        global debug
+        debug = set;
+        
```

Comment:
New line at end of file is missing. Line 24 probably contains spaces or Github is stupid